k8s/test-deployment.py

```
@@ -0,0 +1,437 @@
+                        namespace = doc['metadata'].get('namespace', 'default')
+
+                        key = f"{kind}/{namespace}/{name}"
+                        resources[key] = {
+                            'file': str(file_path),
+                            'kind': kind,
+                            'name': name,
+                            'namespace': namespace,
+                            'spec': doc.get('spec', {}),
+                            'metadata': doc['metadata']
+                        }
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)